app/assets/javascripts/paypal_account_settings.js

```diff
@@ -2,23 +2,28 @@ window.ST = window.ST || {};
 
 (function(module) {
 
-  module.initializePayPalAccountForm = function(formId) {
-    var form = $('#'+formId);
+
+  module.initializeNewPayPalAccountHandler = function(button_id, action) {
```

Comment:
Too much Ruby makes you write snake_cases in JavaScript, although the way to go in JS is camelCase